pkgs/top-level/all-packages.nix

```diff
@@ -39617,6 +39617,9 @@ with pkgs;
   vite = libsForQt5.callPackage ../applications/science/misc/vite { };
 
   xearth = callPackage ../applications/science/astronomy/xearth { };
+
+  xephem = callPackage ../applications/science/astronomy/xephem { };
```

Comment:
Consider moving `pkgs/applications/science/astronomy/xephem/default.nix` to `pkgs/by-name/xe/xephem/package.nix`, so you don't need this line in all-packages.nix.